Makefile

```diff
@@ -50,7 +51,7 @@ testacc:
 .PHONY: testacc
 
 testacc-dev:
-	./scripts/testacc-dev $(TESTS) $(LOG_LEVEL)
+	./scripts/testacc-dev $(TESTS) $(LOG_LEVEL) $(SWEEP)
```

Comment:
TODO: pending the approach we land on, we'll want to add a scheduled CI action to run the sweeper at a given interval (such as overnight when things are quiet).